### DISCUSSION TITLE
Linux performances issues CGO

### DISCUSSION THREAD

**Discussion by Julien2313** [hajimehoshi/ebiten#2674] [Q&A]
Hello,
On my VM on Linux from a Windows host, I have issue on performances directly linked to the size of the windows. The biggest it is, the less FPS I get (TPS are still at 60) while on windows I keep having 144FFS (I have 2 G-Sync monitors)

For the CPU usage, it's around 45% with Linux, 10% on Windows.
Here is on Linux the pproof chart on an example with an empty Draw() and Update():
![image](https://github.com/hajimehoshi/ebiten/assets/8778771/1b049c76-8044-46f4-8aad-95094e2796b5)

I found [this issue](https://github.com/hajimehoshi/ebiten/issues/1130), which looks like to be the same issue as me ? (for the cgo part)
I haven't really find (maybe badly searched) others issues, and so no solution... I don't know if you might know how to solve this, or if I'm doomed ?

Thank you for your time!
Julien

**Comment by hajimehoshi** [comment]
Usually, the overhead of Cgo is the heaviest task in most applications and your application's state seems healthy. So, I think this is an issue in your graphics drivers in the VM, but I am not 100% sure. Would it be possible to test a Linux machine without a VM?
